### PR TITLE
Modify RPCs to make copying and pasting easier to users

### PR DESCRIPTION
**Pull request by TrustHenry**
메타마스크 네트워크 등록시 사용자가 편하게 복사 붙여넣기 할 수 있도록 지원

### CODE REVIEW

File: source/kr/biznet/wallet/tutorials/metamask.md [hunk 1]
```diff
@@ -59,17 +59,17 @@
...
 
-* Mainnet
-      * [RPC URL](./../../develop/rpc.md)
-      * 체인 ID: 0x867 (2151 in decimal)
+* 테스트넷
+      * https://testnet.bosagora.org  [RPC URL](./../../develop/rpc.md)
+      * 체인 ID: 2019 (0x7E3)
       * 통화기호: BOA
-      * [블록탐색기 URL](https://scan.bosagora.org)
+      * https://testnet-scan.bosagora.org  [블록탐색기 URL](https://testnet-scan.bosagora.org)
```
Comment (MichaelKim20) [inline source/kr/biznet/wallet/tutorials/metamask.md]: * 블록탐색기 URL: [https://testnet-scan.bosagora.org](https://testnet-scan.bosagora.org)

File: source/en/biznet/wallet/tutorials/metamask.md [hunk 2]
```diff
@@ -57,17 +57,17 @@ Congratulations! you have creat your MetaMask account!
...
 
-* Mainnet
-      * [RPC URLs](./../../develop/rpc.md)
-      * ChainID: 0x867 (2151 in decimal)
+* Testnet BizTestNet
+      * https://testnet.bosagora.org  [RPC URLs](./../../develop/rpc.md)
+      * ChainID: 2019 (0x7E3)
       * Symbol: BOA
-      * [Block Explorer](https://scan.bosagora.org)
+      * https://testnet-scan.bosagora.org [Block Explorer](https://testnet-scan.bosagora.org)
```
Comment (MichaelKim20) [inline source/en/biznet/wallet/tutorials/metamask.md]: * Block Explorer: [https://testnet-scan.bosagora.org](https://testnet-scan.bosagora.org)

**Review by MichaelKim20 — CHANGES_REQUESTED** [review]

File: source/en/biznet/wallet/tutorials/metamask.md [hunk 3]
```diff
@@ -57,17 +57,17 @@ Congratulations! you have creat your MetaMask account!
 
 ![img](assets/metamask08.png)
 
-* Testnet
-      * [RPC URLs](./../../develop/rpc.md)
-      * ChainID: 0x7E3 (2019 in decimal)
+* Mainnet BizNet
```
Comment (MichaelKim20) [inline source/en/biznet/wallet/tutorials/metamask.md]: * BizNet Mainnet

File: source/en/biznet/wallet/tutorials/metamask.md [hunk 4]
```diff
@@ -57,17 +57,17 @@ Congratulations! you have creat your MetaMask account!
...
+      * https://mainnet.bosagora.org  [RPC URLs](./../../develop/rpc.md)
+      * ChainID: 2151 (0x867)
       * Symbol: BOA
-      * [Block Explorer](https://testnet-scan.bosagora.org)
+      * [Block Explorer](https://scan.bosagora.org)
 
-* Mainnet
-      * [RPC URLs](./../../develop/rpc.md)
-      * ChainID: 0x867 (2151 in decimal)
+* Testnet BizTestNet
```
Comment (MichaelKim20) [inline source/en/biznet/wallet/tutorials/metamask.md]: * BizNet Testnet

File: source/en/biznet/wallet/tutorials/metamask.md [hunk 5]
```diff
@@ -57,17 +57,17 @@ Congratulations! you have creat your MetaMask account!
...
+      * https://testnet.bosagora.org  [RPC URLs](./../../develop/rpc.md)
```
Comment (MichaelKim20) [inline source/en/biznet/wallet/tutorials/metamask.md]:
```
[RPC URLs](./../../develop/rpc.md): https://testnet.bosagora.org
```

File: source/en/biznet/wallet/tutorials/metamask.md [hunk 6]
```diff
@@ -57,17 +57,17 @@ Congratulations! you have creat your MetaMask account!
...
 
-* Testnet
-      * [RPC URLs](./../../develop/rpc.md)
-      * ChainID: 0x7E3 (2019 in decimal)
+* Mainnet
+      * [RPC URLs](./../../develop/rpc.md): https://mainnet.bosagora.org
+      * ChainID: 2151 (0x867)
       * Symbol: BOA
-      * [Block Explorer](https://testnet-scan.bosagora.org)
+      * Block Explorer: [Block Explorer](https://scan.bosagora.org)
```
Comment (MichaelKim20) [inline source/en/biznet/wallet/tutorials/metamask.md]: * [Block Explorer](https://scan.bosagora.org): https://scan.bosagora.org

File: source/en/biznet/wallet/tutorials/metamask.md [hunk 7]
```diff
@@ -57,17 +57,17 @@ Congratulations! you have creat your MetaMask account!
...
 
-* Mainnet
-      * [RPC URLs](./../../develop/rpc.md)
-      * ChainID: 0x867 (2151 in decimal)
+* Testnet
+      * [RPC URLs](./../../develop/rpc.md): https://testnet.bosagora.org
+      * ChainID: 2019 (0x7E3)
       * Symbol: BOA
-      * [Block Explorer](https://scan.bosagora.org)
+      * Block Explorer: [Block Explorer](https://testnet-scan.bosagora.org)
```
Comment (MichaelKim20) [inline source/en/biznet/wallet/tutorials/metamask.md]: * [Block Explorer](https://testnet-scan.bosagora.org): https://testnet-scan.bosagora.org

File: source/kr/biznet/wallet/tutorials/metamask.md (hunk 1 above)
Comment (TrustHenry) [reply source/kr/biznet/wallet/tutorials/metamask.md]: Suggestion applied.

File: source/en/biznet/wallet/tutorials/metamask.md (hunk 2 above)
Comment (TrustHenry) [reply source/en/biznet/wallet/tutorials/metamask.md]: Suggestion applied.

File: source/kr/biznet/wallet/tutorials/metamask.md [hunk 8]
```diff
@@ -59,17 +59,17 @@
...
 
-* 테스트넷
-      * [RPC URL](./../../develop/rpc.md)
-      * 체인 ID: 0x7E3 (2019 in decimal)
+* 메인넷
+      * [RPC URL](./../../develop/rpc.md): https://mainnet.bosagora.org  
+      * 체인 ID: 2151 (0x867)
       * 통화기호: BOA
-      * [블록탐색기 URL](https://testnet-scan.bosagora.org)
+      * 블록탐색기 URL: [https://scan.bosagora.org](https://scan.bosagora.org)
```
Comment (MichaelKim20) [inline source/kr/biznet/wallet/tutorials/metamask.md]: * [블록탐색기 URL](https://scan.bosagora.org): https://scan.bosagora.org

File: source/kr/biznet/wallet/tutorials/metamask.md [hunk 9]
```diff
@@ -59,17 +59,17 @@
...
 
-* Mainnet
-      * [RPC URL](./../../develop/rpc.md)
-      * 체인 ID: 0x867 (2151 in decimal)
+* 테스트넷
+      * [RPC URL](./../../develop/rpc.md): https://testnet.bosagora.org
+      * 체인 ID: 2019 (0x7E3)
       * 통화기호: BOA
-      * [블록탐색기 URL](https://scan.bosagora.org)
+      * 블록탐색기 URL: [https://testnet-scan.bosagora.org](https://testnet-scan.bosagora.org)
```
Comment (MichaelKim20) [inline source/kr/biznet/wallet/tutorials/metamask.md]: * [블록탐색기 URL](https://testnet-scan.bosagora.org): https://testnet-scan.bosagora.org

**Review by MichaelKim20 — CHANGES_REQUESTED** [review]

**Review by MichaelKim20 — APPROVED** [review]

**Pull request merged**